app/components/AppLayout.tsx

```diff
@@ -55,6 +55,10 @@ const styles = StyleSheet.create({
   scrollContainer: {
     minHeight: "100%",
   },
+  scrollContainerPreventScrolling: {
+    minHeight: "100%",
+    maxHeight: "100%",
+  },
```

Comment:
okay, sorry for the misleading style name. That was the product from the issue we had with the growing AnswerList body. That style is only enabled while being on chatbot screen to avoid growing out of screen height 100% to always have the next button at the bottom.

I wanted to have a more generic name instead of something related to chatbot. Maybe the style should be named like `scrollContainerPage` or `scrollContainerPageSize`

Reply:
have gone with `scrollContainerPage`